### PR TITLE
Mention that the library requires c++20 in the readme

### PR DESCRIPTION
**Pull request by GiulioRomualdi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Self-contained C++ interface for the [MeshCat visualizer](https://github.com/rde
 
 ## 🏗️ Dependencies
 
-**meshcat-cpp** is a self-contained library. Most dependencies can be cloned at compile time and they are statically linked to the library. The only dependencies you need is a sufficiently recent C++ compiler (full support to C++17), `cmake`, `openssl`, `zlib`, `libuv`, `boost` and `pkg-config`.
+**meshcat-cpp** is a self-contained library. Most dependencies can be cloned at compile time and they are statically linked to the library. The only dependencies you need is a sufficiently recent C++ compiler (full support to C++20), `cmake`, `openssl`, `zlib`, `libuv`, `boost` and `pkg-config`.
```
Comment (traversaro) [inline README.md:24]: Strictly speaking we do not need full support for C++20 (Ubuntu 20.04 works with partial support), but we can also keep the phrase like that to keep it simple.

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
By the way, there is a related badge also at the top of the readme.

**Comment by GiulioRomualdi** [comment]
> By the way, there is a related badge also at the top of the readme.

I updated it, no?

**Comment by traversaro** [comment]
> > By the way, there is a related badge also at the top of the readme.
> 
> I updated it, no?

Ah sorry, I had missed it.

**Pull request merged**